blog/2022-11-02-monitor-airquality-airthings-netdata.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: airquality 
+title: "Monitor indoor air quality with Airthings and Netdata"
+description: "Monitor indoor air quality with Airthings and Netdata"
+tags: [airthings,airquality,indoor,netdata,co2,humidity,temperature,radon,pm1,pm25,voc,pressure]
+keywords: [airthings,airquality,indoor,netdata,co2,humidity,temperature,monitoring,how-to]
+authors: shyam
+---
+
+Monitoring indoor air quality with Airthings and Netdata.
+
+<!--truncate-->
+
+## Indoor air quality and what to monitor
+
+Indoor air quality can be a crucial influence on your health, wellbeing and productivity. 
+
+Understanding and measuring common contaminants and pollutants is the first step towards reducing your risk of air quality health concerns. 
+
+Airthings is a company that makes great air quality sensors that measure a wide variety of different variables including: 
+- **CO2**
+  - Increased Carbon Dioxide levels can ­cause low productivity and infectious disease transmission.
+- **Humidity**
+  - Too much or too little humidity can affect allergies, asthma and cold or flu symptoms.
+- **Temperature**
+  - Indoor temperatures can affect sleep quality as well as performance and mood.
+- **Radon**
+  - An odorless, radioactive gas that can cause cancer with prolonged unmitigated exposure.
+- **Pressure**
+  - Air pressure levels can be a good indicator of whether the ventilation system is working optimally or not.
+- **Particulate Matter (PM1, PM2.5)** 
+  - A mixture of dust, dirt, and liquids that become suspended in the air.
+- **VOC** 
+  - Airborne chemicals and odors emitted from many everyday products, including cleaners, paints and furniture, cosmetics, hobby products, cooking etc.
+
+While Airthings comes with a mobile app - monitoring air quality metrics with Netdata offers some advantages such as being able to monitor it along with other relevant metrics, in-built anomaly detection and alerts. If you have multiple sensors, Netdata is a great option for a composite view. Let's take a look at how to monitor Airthings air quality metrics using Netdata.
+
+## Collecting sensor data
+
+As a pre-requisite you will need:
+- an Airthings device such as the Airthings View Plus which is used in this example. 
+- a Netdata cloud account
+- a system running the Netdata agent (and will need to run the Python collection script)  
+
+1. To start off you will need the API token and client id which can be created at https://dashboard.airthings.com/integrations/api-integration with the scopes ‘'’read:device:current_values’’’ checked.
+
+2. Export the secret key as an environment variable:
+
+```
+ export secret="secret-key"
+```
+
+3. Run this python script (after updating client_id and device_id variables from the Airthings website) as a cronjob piping the results to a csv file 
```

Comment:
Is there supposed to be a link or python script here

Reply:
will link it once the PR gets merged - could help merge it? https://github.com/netdata/community/pull/61

Reply:
done